src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -200,7 +200,8 @@ protected void scrollToElement(WebElement element, int offset) {
     JavascriptExecutor js = (JavascriptExecutor) driver;
     try {
       js.executeScript("var x = $(arguments[0]);"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "var x = $(arguments[0]);" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)